packages/core/src/components/expandable/expandable.tsx

```diff
@@ -14,10 +14,14 @@ import anime, { AnimeInstance } from "animejs";
 import clsx from "clsx";
 import debounce from "debounce";
 
-export interface ExpandableAnimationStartEvent {}
+export interface ExpandableAnimationStartEvent {
```

Comment:
Dit is een beetje bijvangst, ik vond het StartEvent model te mager. Uitgebreid met de staat aan het begin van de animatie en waar de animatie naartoe gaat